src/content/app/tools/blast/components/listed-blast-submission/ListedBlastSubmission.scss

```diff
@@ -49,6 +53,12 @@ and also that it can be of variable width
   margin-left: 1ch;
 }
 
+.showHide {
+  position: absolute;
+  --show-hide-chevron-height: 8px;
+  --show-hide-chevron-margin-left: 15px;
```

Comment:
Why do you need this? You should now be able to just set a class on the ShowHide, with the left margin set.

Reply:
Reverted this change